publish/publish.proj

```diff
@@ -234,7 +234,7 @@
   <Target Name="SetupPublishSymbols" Condition="'$(SymbolServerPath)'!=''" >
     <PropertyGroup>
       <SymbolExpirationInDays Condition="'$(SymbolExpirationInDays)'=='' and '$(SymbolExpirationDate)'==''">1</SymbolExpirationInDays>
-      <ConvertPortablePdbsToWindowsPdbs>true</ConvertPortablePdbsToWindowsPdbs>
+      <ConvertPortablePdbsToWindowsPdbs>false</ConvertPortablePdbsToWindowsPdbs>
```

Comment:
Same as in Arcade publish task. https://github.com/dotnet/arcade/blob/ffc7a871add67b229c7fd41ad1cdff41da926312/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L138